gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/resources/organizations/ReportersResource.java

```diff
@@ -0,0 +1,137 @@
+    private final OrganizationService organizationService;
+
+    @Inject
+    public ReportersResource(ReporterServiceProxy reporterService,
+                             ReporterPluginService reporterPluginService,
+                             OrganizationService organizationService) {
+        this.reporterService = reporterService;
+        this.reporterPluginService = reporterPluginService;
+        this.organizationService = organizationService;
+    }
+
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(summary = "List registered reporters for a security domain",
+            description = "User must have the DOMAIN_REPORTER[LIST] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[LIST] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[LIST] permission on the specified organization. " +
+                    "Except if user has DOMAIN_REPORTER[READ] permission on the domain, environment or organization, each returned reporter is filtered and contains only basic information such as id and name and type.")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "List registered reporters for a security domain",
+                    content = @Content(mediaType = "application/json",
+                            array = @ArraySchema(schema = @Schema(implementation = Reporter.class)))),
+            @ApiResponse(responseCode = "500", description = "Internal server error")})
+    public void list(
+            @PathParam("organizationId") String organizationId,
+            @Suspended final AsyncResponse response) {
+
+        User authenticatedUser = getAuthenticatedUser();
+
+        checkPermission(ReferenceType.ORGANIZATION, organizationId, Permission.ORGANIZATION_REPORTER, Acl.LIST)
+                .andThen(reporterService.findByReference(Reference.organization(organizationId))
+                        .toList()
+                )
+                .flatMap(reporters ->
+                        hasPermission(authenticatedUser, ReferenceType.ORGANIZATION, organizationId, Permission.ORGANIZATION_REPORTER, Acl.READ)
+                                .map(hasPermission -> reporters.stream()
+                                        .map(reporter -> reporter.apiRepresentation(!hasPermission))
+                                        .toList())
+                )
+                .subscribe(response::resume, response::resume);
+    }
+
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Operation(summary = "Create a reporter for a security domain",
+            description = "User must have the DOMAIN_REPORTER[CREATE] permission on the specified domain " +
+                    "or DOMAIN_REPORTER[CREATE] permission on the specified environment " +
+                    "or DOMAIN_REPORTER[CREATE] permission on the specified organization.")
+    @ApiResponses({
```

Comment:
TODO: permission shouldn't be related to the domain or env here as we are at Org level, this is OK into the code but this description needs to be changed